build failed with error

In top of the tree, "make build" failed with the following error
```
lipingx-m01:docker-volume-vsphere lipingx$ make build-all
/Library/Developer/CommandLineTools/usr/bin/make --directory=client_plugin build-all
../misc/scripts/build.sh pylint
make: Entering directory '/go/src/github.com/vmware/docker-volume-vsphere/esx_service'
make: Nothing to be done for 'pylint'.
make: Leaving directory '/go/src/github.com/vmware/docker-volume-vsphere/esx_service'
../misc/scripts/build.sh build
docker: Error response from daemon: invalid bind mount spec "/Users/lipingx/.ssh/id_rsa::ro": invalid volume specification: '/Users/lipingx/.ssh/id_rsa::ro': invalid mount config for type "bind": field Target must not be empty.
See 'docker run --help'.
make[1]: *** [dockerbuild] Error 125
make: *** [build-all] Error 2
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -110,6 +110,11 @@ The prerequisite to build and deploy the plugin is to have a DockerHub account. 
 
 - The build will use your `username` (the output of `whoami`) to decide on the `DOCKER_HUB_REPO` name to complete our move to use managed plugin. If you want to use another DockerHub repository you need to set `DOCKER_HUB_REPO` as environment variable.
 
+- You **need** to set enviroment variable `SSH_KEY_OPT` if your ssh keys are not stored in the default location with the following format:
+```
+export SSH_KEY_OPT='-i /Users/username/.ssh/id_rsa'
+```
+
 - Test verification is extended using govmomi integration and `govc` cli is **required to set** following environment variables.
   - `GOVC_USERNAME` & `GOVC_PASSWORD`: user credentials for logging in to `ESX IP`
 
@@ -363,7 +368,7 @@ Head to GitHub and author a new release add the changelog for the tag created.
 <vibID>vmware-esx-vmdkops-0.15.b93c186</vibID> (vmware-esx-vmdkops-<release_version>.<commit's SHA hash>)
 ```
 
-4. After the vib signing has successfully finished, you will receive a confirmation email. Go to the deliverables link 
+4. After the vib signing has successfully finished, you will receive a confirmation email. Go to the deliverables link
    mentioned in the email and download the signed offline bundle. Offline bundle will be a zip file something like	```vDVS_driver-0.18-offline_bundle-7032243.zip```. From the offline bundle, extract the signed VIB.
 5. Head to [Bintray](https://bintray.com/vmware/product/vDVS/view) to publish signed VIB as well as offline bundle
 6. Push vDVS & vFile managed plugin to docker store and windows plugin to bintray

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -110,7 +110,7 @@ The prerequisite to build and deploy the plugin is to have a DockerHub account. 
 
 - The build will use your `username` (the output of `whoami`) to decide on the `DOCKER_HUB_REPO` name to complete our move to use managed plugin. If you want to use another DockerHub repository you need to set `DOCKER_HUB_REPO` as environment variable.
 
-- You **need** to set enviroment variable `SSH_KEY_OPT` if your ssh keys are not stored in the default location with the following format:
+- You **need** to set environment variable `SSH_KEY_OPT` if your ssh keys are not stored in the default location with the following format:
 ```
 export SSH_KEY_OPT='-i /Users/username/.ssh/id_rsa'
 ```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -113,6 +113,7 @@ The prerequisite to build and deploy the plugin is to have a DockerHub account. 
 - You **need** to set environment variable `SSH_KEY_OPT` if your ssh keys are not stored in the default location with the following format:
 ```
 export SSH_KEY_OPT='-i /Users/username/.ssh/id_rsa'
+
 ```
 
 - Test verification is extended using govmomi integration and `govc` cli is **required to set** following environment variables.

--- a/client_plugin/drivers/vfile/kvstore/etcdops/etcdops.go
+++ b/client_plugin/drivers/vfile/kvstore/etcdops/etcdops.go
@@ -23,6 +23,7 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
@@ -132,8 +133,10 @@ func NewKvStore(dockerOps *dockerops.DockerOps) *EtcdKVS {
 		}
 		// when error is IsNotExist, ETCD data-dir is not existing,
 		// need to continue to create/join a new ETCD cluster
+		log.Infof("ETCD data-dir does not exist, continue to create/join a new ETCD cluster")
 	} else {
 		// ETCD data-dir already exists, just re-join
+		log.Infof("ETCD data-dir exists, continue to rejoin")
 		err = e.rejoinEtcdCluster()
 		if err != nil {
 			log.Errorf("Failed to rejoin the ETCD cluster: %v", err)
@@ -190,6 +193,7 @@ func NewKvStore(dockerOps *dockerops.DockerOps) *EtcdKVS {
 func (e *EtcdKVS) rejoinEtcdCluster() error {
 	nodeID := e.nodeID
 	nodeAddr := e.nodeAddr
+	log.Infof("rejoinEtcdCluster on node with nodeID %s and nodeAddr %s", nodeID, nodeAddr)
 	lines := []string{
 		"--name", nodeID,
 		"--data-dir", etcdDataDir,
@@ -216,8 +220,10 @@ func (e *EtcdKVS) startEtcdCluster() error {
 	nodeAddr := e.nodeAddr
 	log.Infof("startEtcdCluster on node with nodeID %s and nodeAddr %s", nodeID, nodeAddr)
 
+	files, err := filepath.Glob(etcdDataDir)
+	log.Debugf("Files in etcdDataDir: %s", files)
 	// create ETCD data directory with 755 permission since only root needs full permission
-	err := os.Mkdir(etcdDataDir, 0755)
+	err = os.Mkdir(etcdDataDir, 0755)
 	if err != nil {
 		log.Errorf("Failed to create directory etcd-data: err %v", err)
 		return err

--- a/misc/scripts/build.sh
+++ b/misc/scripts/build.sh
@@ -123,7 +123,7 @@ then
   $DOCKER run --rm  -v $PWD/..:$dir -w $dir $pylint_container $MAKE_ESX pylint
 else
   docker_socket=/var/run/docker.sock
-  if [ -z '$SSH_KEY_OPT' ]
+  if [ -z "$SSH_KEY_OPT" ]
   then
     SSH_KEY_OPT="-i /root/.ssh/id_rsa"
   fi

